release-notes.md

```diff
@@ -0,0 +1,26 @@
+# Release 3.0.0
+
+- [linpad] added support for LINQPad 6 including new [nuget package](https://www.nuget.org/packages/linq2db.LINQPad)  ([#29](https://github.com/linq2db/linq2db.LINQPad/issues/33))
+- [tools] driver updated to use recent linq2db 3.0.1 release
+- [tools] switched to [Humanizer.Core](https://www.nuget.org/packages/Humanizer.Core) for model pluralization
+- [providers] removed ODP.NET Native Oracle provider (we still support managed provider, which doesn't require client installation)
+- [providers] added support for MS Access ODBC provider
+- [providers] replaced MySql.Data provider with [MySqlConnector](https://www.nuget.org/packages/MySqlConnector) provider
+- [providers] replaced native SAP/Sybase ASE provider with managed [AdoNetCore.AseClient](https://www.nuget.org/packages/AdoNetCore.AseClient) provider
+- [providers] removed IBM DB2 iSeries support temporary because required [provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries) lacks linq2db v3 support
```

Comment:
```suggestion
- [providers] removed IBM DB2 iSeries support temporarily because required [provider](https://github.com/LinqToDB4iSeries/Linq2DB4iSeries) lacks linq2db v3 support
```